pages/about.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://kit.fontawesome.com/a1e854484d.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet" />
+    <link rel="stylesheet" href="../dist/css/style.css" />
+    <script defer src="../app/js/script.js"></script>
+    <title>Africa's Leading Conference 2023 | About Page</title>
+  </head>
+  <body class="about">
+    <header class="header">
+      <section class="header__nav-section">
+        <div class="container container-px">
+          <nav class="header__nav flex-jc-sb flex-ai-c">
+            <button id="btnHumburger" class="header__hamburger flex fle-jc-c flex-ai-c flex-collumn">
+              <span class="header__hamburger-stroke header__hamburger-top"></span>
+              <span class="header__hamburger-stroke header__hamburger-middle"></span>
+              <span class="header__hamburger-stroke header__hamburger-bottom"></span>
+            </button>
+
```

Comment:
![image](https://user-images.githubusercontent.com/60612329/210952745-7493bb37-7abf-4372-8739-7359d0f0a4d3.png)
on your hamburger menu in the about section of your page's mobile version, your home link takes you to nowhere and returns an error kindly fix that please 👍🏾

Reply:
I got you @alexander16108

I have just fixed that.

Reply:
Fixed.